guide/14-deep-learning/leveraging_llms_for_real_time_decision_making_in_calamities.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
```

Comment:
<p>The ESRI Model Definition (.emd) file will include both required and optional keys to facilitate model execution. To run the&nbsp;<strong>Classify Text Using Deep Learning (GeoAI)</strong>&nbsp;tool, we need to supply the&nbsp;<code>InferenceFunction</code>,&nbsp;<code>ModelType</code>, and&nbsp;<code>OutputField</code>.</p><p> </p><p>Other keys are the prerogative of the Model extension author. In this case, we utilized <code>few-shot prompting</code>&nbsp; by providing the following information:&nbsp;<code>examples</code>&nbsp;and&nbsp;<code>prompt</code>. We will utilize this information to construct a clear and effective prompt for the task.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>updated</p>